proof/refine/ARM/Invariants_H.thy

```diff
@@ -1010,15 +1016,18 @@ definition "vs_valid_duplicates' \<equiv> \<lambda>h.
               x && ~~ mask (vs_ptr_align ko) = y && ~~ mask (vs_ptr_align ko) \<longrightarrow>
               h x = h y"
 
+abbreviation
+  "in_reply_queue rptr s \<equiv> replyNexts_of s rptr \<noteq> None \<or> replyPrevs_of s rptr \<noteq> None"
+
```

Comment:
I am happy to have this abbreviation, but I feel that the name might be somewhat misleading.
This is meant to cover the case when there is no SC linked to the queue, right? I.e., the case when the reply at `rptr` is not in the "reply stack", which we sometimes call also as "reply queue". 
Since this statement actually looks only at the adjacent replies, how about something like `is_reply_linked`? (of course, you can then ask linked to what, but...) Or maybe `in_reply_path` as in it is in a `heap_path`.

Reply:
then again, even a stand along reply can be in a `heap_path`. Hmm.

Reply:
Good point with the name. The intention is to cover the case where the reply at `rptr` is linked to other replies, regardless of whether there is an `SC` eventually linked to. I think with that in mind I'm happy to use `is_reply_linked`.